guide/glossary.md

```diff
@@ -313,10 +331,6 @@ Many wallet-applications work with HD Wallets and recovery phrases, and are inte
 
 **Technicalities** - Recovery of multisig-wallets needs both the extended public key and the recovery phrase of all paticipating keys as well as the master key fingerprint as defined by BIP32 concatenated with the derivation path of the public key. The derivation path is represented as 32-bit little endian unsigned integer indexes concatenated with each other. The number of 32 bit unsigned integer indexes must match the depth provided in the extended public key.
 
-### Simplified payment verification (SPV)
-
- It is possible to verify bitcoin payments without running a full network node. This is called simplified payment verification, or SPV. A user’s bitcoin spv wallet only needs a copy of the block headers of the longest chain, which are available by querying network nodes until it is apparent that the longest chain has been obtained. SPV lets you validate your transactions without having to worry about anybody else’s transactions. It ensures your transactions are in a block, and it provides confirmations that additional blocks are being added to the chain. An SPV wallet is a type of bitcoin wallet that works this way.
-
 **References:**
 
 - [WabiSabi](https://github.com/zkSNACKs/WabiSabi/blob/master/explainer.md)
```

Comment:
Unrelated to this PR, but noticed that this reference is for CoinJoin info but placed below the SPV paragraph. We should address that in a separate PR.

Reply:
Opened a PR for this https://github.com/BitcoinDesign/Guide/pull/373